runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/cache/KafkaCacheCursorFactory.java

```diff
@@ -71,7 +72,6 @@ public KafkaCacheCursorFactory(
     {
         this.writeBuffer = writeBuffer;
         this.checksum = new CRC32C();
-        this.nullKeyInfo = initNullKeyInfo(checksum);
```

Comment:
I think you might have misinterpreted the feedback. 🙂 

I was saying that we want to preserve the original optimization of not copying `null` keys all the time, just the approach is slightly different now because we cannot pre-create the entire filter condition.